core/os/dir_bsd.odin

```diff
@@ -3,7 +3,7 @@ package os
 
 import "core:mem"
 
-read_dir :: proc(fd: Handle, n: int, allocator := context.allocator) -> (fi: []File_Info, err: Errno) {
+read_dir :: proc(fd: Handle, fi_size: int = 100, allocator := context.allocator) -> (fi: []File_Info, err: Errno) {
```

Comment:
Please no. If it was confusing, you should have improved the documentation, not add a default parameter.